tiledb/sm/query/readers/sparse_global_order_reader.cc

```diff
@@ -198,6 +202,8 @@ Status SparseGlobalOrderReader<BitmapType>::dowork() {
   do {
     stats_->add_counter("loop_num", 1);
 
+    load_tile_offsets_for_fragments(subarray_.relevant_fragments());
```

Comment:
Are we calling this again because `relevant_fragments` on the subarray object is being mutated somewhere in the loop? (first call above via `load_all_tile_offsets`)

Reply:
Good catch... removed!